### DISCUSSION TITLE
end to end encryption details

### DISCUSSION THREAD

**Discussion by praveensounder** [nginx/kubernetes-ingress#2863] [Q&A] [answered]
Is it possible to have secure backend connections from the nginx controller.

How does the communication is happening?

**Comment by brianehlert** [comment, marked as answer]
Encryption from the NGINX Ingress Controller to the backends can be achieved using the egressMTLS policy.
https://docs.nginx.com/nginx-ingress-controller/configuration/policy-resource/#egressmtls

Simple example is here: https://github.com/nginxinc/kubernetes-ingress/tree/v2.3.0/examples/custom-resources/egress-mtls
(this resource has not changed for a few versions now)
